### PR TITLE
Add log lines of update history in the scripts for WAFS processing enhancement

### LINKED ISSUE #334: Adding change logs for GFS V16.1.2 Implementation
The UPP branch release/gfs_v16 are updated on workflow scripts for wafs post processing enhancement and triggering postanl job for reliability for GFS V16.1.2 implementation. NCO requests adding change logs in the relevant scripts.

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
The main changes are:
1) Add change logs in relevant workflow scripts for wafs processing update.
2) Drop the fix of triggering post_anl job for reliability due to concern of GFS operation.

### CODE REVIEW

File: scripts/exglobal_atmos_pmgr.sh [hunk 1]
```diff
@@ -71,7 +73,7 @@ do
 
   sleep 10
   icnt=$((icnt + 1))
-  if [ $icnt -ge 1080 ]
+  if [ $icnt -ge 1001 ]
```
Comment (WenMeng-NOAA) [inline scripts/exglobal_atmos_pmgr.sh]: @lgannoaa May I ask the reason of changing 1080 into 1001? I recalled the 1080 was added based on NCO GFS V16 testing. @KateFriedman-NOAA may chime in.

**Comment by lgannoaa** [comment]
Hi Wen,

Arun authorized task is to add comments to the file.
Please only pick up the comments session from my file.

Lin

File: scripts/exglobal_atmos_pmgr.sh (hunk 1 above)
Comment (WenMeng-NOAA) [reply scripts/exglobal_atmos_pmgr.sh]: @lgannoaa Got it. Thanks for clarifying.

**Comment by WenMeng-NOAA** [comment]
@YaliMao-NOAA Please commit the changes of comment part in exglobal_atmos_pmgr.sh only as:
#  Script name:         exglobal_atmos_pmgr.sh
#
#  This script monitors the progress of the gfs_post job
#  Script history log:
#  2021-05-25   Lin Gan: Ensure loganl.txt file created by atmos_analysis_calc before release atmos_post_anl job.
#

**Comment by YaliMao-NOAA** [comment]
@WenMeng-NOAA exglobal_atmos_pmgr.sh now only includes the changes of comment part.

**Review by WenMeng-NOAA — APPROVED** [review]
Look good to me.

**Comment by WenMeng-NOAA** [comment]
@YaliMao-NOAA They look good to me. Thanks!

**Review by WenMeng-NOAA — APPROVED** [review]

**Pull request merged**